nix/overlay.nix

```diff
@@ -81,4 +81,8 @@ in {
   aapt2 = callPackage ./pkgs/aapt2 { };
   patchMavenSources = callPackage ./pkgs/patch-maven-srcs { };
   goMavenResolver = callPackage ./pkgs/go-maven-resolver { };
+  watchman = callPackage ./pkgs/watchman {
```

Comment:
I would move that higher, next to `git-lfs` and `ios-deploy`, since this is a fix for an issue, and not just a new package.

Also, a comment explaining why we are doing this would be good too.